typeGenAutoConfig wrong relative path in Windows

These lines generate the imports of the typegenAutoConfig. https://github.com/prisma/nexus/blob/18a306ee2903d98fc110c1067e1903ea0ee5f256/src/typegenAutoConfig.ts#L212-L215  And use this function https://github.com/prisma/nexus/blob/18a306ee2903d98fc110c1067e1903ea0ee5f256/src/typegenAutoConfig.ts#L340-L350

The problem is with this line:
```ts
    return `.${path.sep}${path.join(relative, filename)}`;
```

In Windows it generates an import statement with backslashes `\`, and when node tries to compile it, it can't.

Maybe change it to:
```ts
    return `./${path.join(relative, filename)}`;
```

When node is trying to parse the string, it assumes the backslash is escape character, so what I get is:

```ts
src/generated/nexus.ts(6,24): error TS2307: Cannot find module '..interfacescontext'.
```

--- a/src/typegenAutoConfig.ts
+++ b/src/typegenAutoConfig.ts
@@ -321,8 +321,9 @@ export function typegenAutoConfig(options: TypegenAutoConfigOptions) {
 
     Object.keys(importsMap).sort().forEach((alias) => {
       const [importPath, glob] = importsMap[alias];
+      const safeImportPath = importPath.replace(/\\+/g, '/');
       imports.push(
-        `import ${glob ? "* as " : ""}${alias} from "${importPath}"`
+        `import ${glob ? "* as " : ""}${alias} from "${safeImportPath}"`
       );
     });
 

